### PR TITLE
[Documentation] Update

### PR DESCRIPTION
**Pull request by bugwelle**

### CODE REVIEW

File: Documentation/SAD/SAD.md [hunk 1]
```diff
@@ -49,14 +49,22 @@ This document describes the architecture of the WGPlaner project.
...
-| [UC Pinboard](../UC/UC_Pinboard.md)                                | 2017-11-26 | WGPlaner Team             |
-| [UC Registration](../UC/UC_Registration.md)                        | 2017-11-26 | WGPlaner Team             |
-| [UC Shopping List](../UC/UC_Shopping_List.md)                      | 2017-11-26 | WGPlaner Team             |
+| [GitHub - Server](http://github.com/WGPlaner/wg_planer_server)     | 2018-06-11 | WGPlaner Team             |
+| [GitHub - Client](http://github.com/WGPlaner/wg_planer)            | 2018-06-11 | WGPlaner Team             |
 | [UC Add Item](../UC/UC_Add_Item.md)                                | 2017-12-06 | WGPlaner Team             |
 | [UC Buy Item](../UC/UC_Buy_Item.md)                                | 2017-12-06 | WGPlaner Team             |
+| [UC Invite New Group Member](../UC/UC_Invite_New_Group_Member.md)  | 2017-12-06 | WGPlaner Team             |
+| [UC List Bought Items](../UC/UC_List_bought_items.md)              | 2018-06-11 | WGPlaner Team             |
+| [UC List Fixed Costs](../UC/UC_List_fixed_costs.md)                | 2018-06-11 | WGPlaner Team             |
```
Comment (ArneSchulze) [inline Documentation/SAD/SAD.md]: not ready

File: Documentation/SAD/SAD.md [hunk 2]
```diff
@@ -49,14 +49,22 @@ This document describes the architecture of the WGPlaner project.
...
+| [UC List Received Bills](../UC/UC_List_received_bills.md)          | 2018-06-11 | WGPlaner Team             |
+| [UC Pay Bills](../UC/UC_Pay_bills.md)                              | 2018-06-11 | WGPlaner Team             |
```
Comment (ArneSchulze) [inline Documentation/SAD/SAD.md]: not ready

File: Documentation/SAD/SAD.md [hunk 3]
```diff
@@ -49,14 +49,22 @@ This document describes the architecture of the WGPlaner project.
...
+| [UC Pinboard](../UC/UC_Pinboard.md)                                | 2017-11-26 | WGPlaner Team             |
```
Comment (ArneSchulze) [inline Documentation/SAD/SAD.md]: not ready

File: Documentation/TestPlan/TestPlan.md [hunk 4]
```diff
@@ -34,24 +34,26 @@ This test plan is intended for technically advanced readers and developers. It d
...
+| Title                                                              | Date       | Publishing organization   |
+| -------------------------------------------------------------------|:----------:| ------------------------- |
+| [WGPlaner Blog](http://wgplanerblog.wordpress.com/)                | 2017-11-26 | WGPlaner Team             |
+| [GitHub - Server](http://github.com/WGPlaner/wg_planer_server)     | 2018-06-11 | WGPlaner Team             |
+| [GitHub - Client](http://github.com/WGPlaner/wg_planer)            | 2018-06-11 | WGPlaner Team             |
+| [UC Add Item](../UC/UC_Add_Item.md)                                | 2017-12-06 | WGPlaner Team             |
+| [UC Buy Item](../UC/UC_Buy_Item.md)                                | 2017-12-06 | WGPlaner Team             |
+| [UC Invite New Group Member](../UC/UC_Invite_New_Group_Member.md)  | 2017-12-06 | WGPlaner Team             |
+| [UC List Bought Items](../UC/UC_List_bought_items.md)              | 2018-06-11 | WGPlaner Team             |
+| [UC List Fixed Costs](../UC/UC_List_fixed_costs.md)                | 2018-06-11 | WGPlaner Team             |
```
Comment (ArneSchulze) [inline Documentation/TestPlan/TestPlan.md]: same

File: Documentation/TestPlan/TestPlan.md [hunk 5]
```diff
@@ -34,24 +34,26 @@ This test plan is intended for technically advanced readers and developers. It d
...
+| [UC List Received Bills](../UC/UC_List_received_bills.md)          | 2018-06-11 | WGPlaner Team             |
+| [UC Pay Bills](../UC/UC_Pay_bills.md)                              | 2018-06-11 | WGPlaner Team             |
```
Comment (ArneSchulze) [inline Documentation/TestPlan/TestPlan.md]: same

File: Documentation/TestPlan/TestPlan.md [hunk 6]
```diff
@@ -34,24 +34,26 @@ This test plan is intended for technically advanced readers and developers. It d
...
+| [UC Pinboard](../UC/UC_Pinboard.md)                                | 2017-11-26 | WGPlaner Team             |
```
Comment (ArneSchulze) [inline Documentation/TestPlan/TestPlan.md]: same

File: Documentation/SAD/SAD.md (hunk 1 above)
Comment (bugwelle) [reply Documentation/SAD/SAD.md]: done

File: Documentation/SAD/SAD.md (hunk 2 above)
Comment (bugwelle) [reply Documentation/SAD/SAD.md]: done

File: Documentation/SAD/SAD.md (hunk 3 above)
Comment (bugwelle) [reply Documentation/SAD/SAD.md]: done

File: Documentation/TestPlan/TestPlan.md (hunk 4 above)
Comment (bugwelle) [reply Documentation/TestPlan/TestPlan.md]: done

File: Documentation/TestPlan/TestPlan.md (hunk 6 above)
Comment (bugwelle) [reply Documentation/TestPlan/TestPlan.md]: done

File: Documentation/TestPlan/TestPlan.md (hunk 5 above)
Comment (bugwelle) [reply Documentation/TestPlan/TestPlan.md]: done

**Pull request merged**